app/services/scrapers/scraper_manager.py

```
@@ -12,9 +13,12 @@ class ScraperManager:
 
     bluesky_scraper: Optional[BlueskyScraper] = None
     weibo_scraper: Optional[WeiboScraper] = None
+    firecrawl_scraper: Optional[FirecrawlScraper] = None
 
     scrapers = {"bluesky": bluesky_scraper,
-                "weibo": bluesky_scraper}
+                "weibo": weibo_scraper,
+                "other": firecrawl_scraper,
+                "unknown": firecrawl_scraper}
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)